### PR TITLE
Search sighting front

### PR DESCRIPTION
**Pull request by callumfg**
Search sighting frontend

### CODE REVIEW

File: ClientApp/src/Api/apiClient.ts [hunk 1]
```diff
@@ -41,6 +41,18 @@ export interface SightingResponse {
...
 
+export interface ListResponse<T> {
+  items: T[];
+  // totalNumberOfItems: number;
+  // page: number;
+  // nextPage: string;
+  // previousPage: string;
+}
+
+
```
Comment (chloejonessoftwire) [inline ClientApp/src/Api/apiClient.ts:52]: quite a lot of empty lines here

File: ClientApp/src/Pages/SearchSightings/SearchSightings.tsx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+        setFormStatus("SUBMITTING");
+        submitSearch(
+          species,
+          location,
+          sightedAt 
+        ).then((data) => setSearchResults(data))
+          .catch(() => setFormStatus("ERROR"))
+          .then(() => setPageStatus("RESULTS"))
+          .then(() => setFormStatus("READY"));
+
```
Comment (chloejonessoftwire) [inline ClientApp/src/Pages/SearchSightings/SearchSightings.tsx:45]: again, empty space

File: ClientApp/src/Pages/SearchSightings/SearchSightings.tsx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+            </button>
+            {formStatus === "ERROR" && <p>Something went wrong! Please try again.</p>}
+          </form>
+        </div>
+      );
+
+
+
+
+
```
Comment (chloejonessoftwire) [inline ClientApp/src/Pages/SearchSightings/SearchSightings.tsx]: lots of empty space :)

**Review by chloejonessoftwire — COMMENTED** [review]
sorry for being pedantic, there are quite a few empty lines that you might want to delete to neated up the code. otherwise looks good ! :D

File: ClientApp/src/Pages/SearchSightings/SearchSightings.tsx (hunk 3 above)
Comment (littlemauli) [reply ClientApp/src/Pages/SearchSightings/SearchSightings.tsx]: I agree with Chloe on the empty spaces

**Review by littlemauli — COMMENTED** [review]
it is so neat! just a few empty spaces.

File: ClientApp/src/Pages/SearchSightings/SearchSightings.scss [hunk 4]
```diff
@@ -0,0 +1,27 @@
+@import "../../Styles/constants";
+@import '../../Styles/styles';
+
+
+.search-sighting-page {
+    margin: $gutter-width;
+    height: 100%;
+  }
+  
+  .search-sighting-form {
```
Comment (ChristianMcc) [inline ClientApp/src/Pages/SearchSightings/SearchSightings.scss:10]: all these styles are a bit indented I think

File: ClientApp/src/Pages/SearchSightings/SearchSightings.tsx [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+              type="submit"
+            >
+              Search
+            </button>
+            {formStatus === "ERROR" && <p>Something went wrong! Please try again.</p>}
+          </form>
+        </div>
+        
+      );
+
```
Comment (ChristianMcc) [inline ClientApp/src/Pages/SearchSightings/SearchSightings.tsx:162]: some spare white space

**Review by ChristianMcc — APPROVED** [review]
Looks great! couple of formatting tweaks - have approved so you can merge when those are done without another review :)

**Pull request merged**